clients/java/src/main/java/io/camunda/client/CamundaClientBuilder.java

```diff
@@ -232,6 +233,17 @@ CamundaClientBuilder jobWorkerExecutor(
    */
   CamundaClientBuilder preferRestOverGrpc(final boolean preferRestOverGrpc);
 
+  /**
+   * A custom retryBackoffSupplier allows the client to determine the default retry backoff strategy
+   * that every job worker will apply unless configured otherwise. By default, a static retry
+   * backoff of <code>PT0S</code> is used.
+   *
+   * @param jobExceptionHandler the retry backoff supplier to retrieve the retry backoff for every
+   *     fail job command
+   * @return this builder for chaining
+   */
+  CamundaClientBuilder defaultJobWorkerExceptionHandler(JobExceptionHandler jobExceptionHandler);
```

Comment:
💭 This is not the default right ? This, if provided, is a custom handler provided by the user. The name is misleading in my opinion.

Reply:
This is the default as the exception handler can be set to each individual job worker.

Reply:
Yep, but it's not the default provided by us. It's the general handler that the user can provide. wdyt about `globalJobWorkerExceptionHandler` ?